packages/playwright-core/src/server/dispatchers/localUtilsDispatcher.ts

```diff
@@ -40,6 +40,7 @@ import type http from 'http';
 import type { Playwright } from '../playwright';
 import { SdkObject } from '../../server/instrumentation';
 import { serializeClientSideCallMetadata } from '../../utils';
+import { deviceDescriptors as descriptors }  from '../deviceDescriptors';
```

Comment:
I guess we can just import from the json file directly and remove `deviceDescriptors.ts` entirely?

Reply:
In this specific file, it works, but in other files it complains because it cannot infer it's a `Devices` type. By having json content exposed in `deviceDescriptors.ts` as `Devices` we no longer have to cast it in other files.

Reply:
BTW, with `require('../deviceDescriptorsSource.json')`, inferred type was `any`:

![image](https://github.com/microsoft/playwright/assets/1374559/c6152c0d-5322-4d45-8f66-eee19f44b180)